### PR TITLE
NGSTACK-464 Prime translations bundle

### PR DESCRIPTION
**Pull request by RandyCupic**
Required changes:

Create database tables:
`php bin/console doctrine:schema:update --force`

Import existing translations into database:
`php bin/console lexik:translations:import AppBundle`

Install assets:
`php bin/console assets:install --symlink`

Clear caches:
`php bin/console cache:clear`

### CODE REVIEW

File: app/config/config.yml [hunk 1]
```diff
@@ -191,3 +191,13 @@ httplug:
         default:
             factory: httplug.factory.curl
             plugins: [httplug.plugin.logger]
+
+lexik_translation:
+    fallback_locale: [en]
+    managed_locales: [en]
+
+    base_layout: "PrimeTranslationsBundle::pagelayout.html.twig"
```
Comment (emodric) [inline app/config/config.yml:199]: Needs to be referenced with Twig notation: `@PrimeTranslations/pagelayout.html.twig`

File: composer.json [hunk 2]
```diff
@@ -83,7 +83,8 @@
         "netgen/layouts-ezplatform": "~1.2.0",
         "netgen/layouts-ezplatform-site-api": "~1.2.0",
         "netgen/layouts-ezplatform-relation-list-query": "^1.2",
-        "netgen/layouts-ezplatform-tags-query": "^1.2"
+        "netgen/layouts-ezplatform-tags-query": "^1.2",
+        "primedigital/translations-bundle": "^1.0"
```
Comment (emodric) [inline composer.json:87]: Can you move this directly below `netgen/admin-ui-bundle` as I'd like to keep this section separate for Netgen Layouts?

File: app/config/config.yml (hunk 1 above)
Comment (RandyCupic) [reply app/config/config.yml]: Fixed in https://github.com/netgen/media-site/pull/32/commits/30eaaea629b84d9c57c4b41c3f2831f7207b244d

File: composer.json (hunk 2 above)
Comment (RandyCupic) [reply composer.json]: Fixed in https://github.com/netgen/media-site/pull/32/commits/30eaaea629b84d9c57c4b41c3f2831f7207b244d

**Comment by emodric** [comment]
Thanks @RandyCupic !

**Pull request merged**